less/responsive.less

```diff
@@ -45,4 +45,4 @@
 // ------------------
 
 // From 979px and below, show a button to toggle navbar contents
-@import "responsive-navbar.less";
+//@import "responsive-navbar.less";
```

Comment:
Yeah was wondering whether to just remove it - thought if we wanted to bring it back it might be useful to see what should be included.